RELEASE_NOTES.md

```diff
@@ -2,6 +2,20 @@
 
 ## Unreleased
 
+## 3.0.0 (2016-02-07)
+
+#### BREAKING CHANGES
+* `git hack`: no longer accepts a parent branch
```

Comment:
I guess this [comment](https://github.com/Originate/git-town/pull/778#issuecomment-269514502) is the where it started